### DISCUSSION TITLE
Dunes of Terror

### DISCUSSION THREAD

**Discussion by customautosys** [melonjs/melonJS#1224] [Show and tell]
Public beta of my new game, made with Quasar and Melonjs! https://play.google.com/store/apps/details?id=com.customautosys.dunes_of_terror

This is a board game which is a battle between the Police Special Forces (Blue Team) and Terrorist Gangster Criminals (Red Team). There are a variety of weapons to attack the enemies with. Currently supports AI and same-screen multiplayer.

**Comment by obiot** [comment]
thanks for sharing !

**Comment by obiot** [comment]
Just a first comment : it doesnt work under Safari, by the look of it I would say because you only provide .ogg files for the audio without any mp3 as a backup format.

**Reply by customautosys** [reply]
Just fixed this morning! It now works on my wife's iPad.
